test/unit/lodash.spec.js

```diff
@@ -0,0 +1,568 @@
+            }
+        );
+    });
+
+    describe('lowerFirst', function () {
+        before(function () {
+            params = datasets.lowerFirst();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should return converted string with first letter in lowercase', function () {
+                expect(_.lowerFirst.apply(_, params)).to.eql('sOME STRING TO TEST FUNCTION');
+            }
+        );
+    });
+
+    describe('pad', function () {
+        before(function () {
+            params = datasets.pad();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(Number.isInteger(params[1])).to.eql(true);
+            expect(params[1]).to.be.above(params[0].length);
+            expect(params).to.have.length.within(2, 3);
+            if (params[2] !== undefined && params.length === 3) {
+                expect(params[2]).to.be.a('string');
+            }
+        });
+        it('should return converted string with padding characters', function () {
+                expect(_.pad.apply(_, params)).to.eql('._._.some long sweet text._._.');
+            }
+        );
+    });
+
+    describe('padEnd', function () {
+        before(function () {
+            params = datasets.padEnd();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(Number.isInteger(params[1])).to.eql(true);
+            expect(params[1]).to.be.above(params[0].length);
+            expect(params).to.have.length.within(2, 3);
+            if (params[2] !== undefined && params.length === 3) {
+                expect(params[2]).to.be.a('string');
+            }
+        });
+        it('should return converted string with padding characters on right', function () {
+                expect(_.padEnd.apply(_, params)).to.eql('some another long sweet text.*.*.*.*');
+            }
+        );
+    });
+
+    describe('padStart', function () {
+        before(function () {
+            params = datasets.padStart();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(Number.isInteger(params[1])).to.eql(true);
+            expect(params[1]).to.be.above(params[0].length);
+            expect(params).to.have.length.within(2, 3);
+            if (params[2] !== undefined && params.length === 3) {
+                expect(params[2]).to.be.a('string');
+                expect(params[2]).to.have.length.above(0);
+            }
+        });
+        it('should return converted string with padding characters on left', function () {
+                expect(_.padStart.apply(_, params)).to.eql('|_|_|_|_|_|_try again some long sweet text');
```

Comment:
In my opinion, without looking to this file, that solution is impossible to reveal